src/time/functions/daysToMonths.ts

```diff
@@ -1,82 +1,18 @@
+import { DAYS_IN_MONTHS } from "../../../internal/time/constants/DAYS_IN_MONTHS";
+import { MonthName } from "../../../internal/time/types/MonthName";
 import { Days } from "../types/Days";
 import { Months } from "../types/Months";
 import { Years } from "../types/Years";
 import { isLeapYear } from "./isLeapYear";
 
-export type StartForDaysToMonths = {
-  month:
-    | "January"
-    | "February"
-    | "March"
-    | "April"
-    | "May"
-    | "June"
-    | "July"
-    | "August"
-    | "September"
-    | "October"
-    | "November"
-    | "December";
+export interface StartForDaysToMonths {
+  month: MonthName;
   year: Years;
-};
-
-const DAYS_IN_MONTHS: {
-  day: number;
-  month: StartForDaysToMonths["month"];
-}[] = [
-  {
-    day: 31,
-    month: "January",
-  },
-  {
-    day: 28,
-    month: "February",
-  },
-  {
-    day: 31,
-    month: "March",
-  },
-  {
-    day: 30,
-    month: "April",
-  },
-  {
-    day: 31,
-    month: "May",
-  },
-  {
-    day: 30,
-    month: "June",
-  },
-  {
-    day: 31,
-    month: "July",
-  },
-  {
-    day: 31,
-    month: "August",
-  },
-  {
-    day: 30,
-    month: "September",
-  },
-  {
-    day: 31,
-    month: "October",
-  },
-  {
-    day: 30,
-    month: "November",
-  },
-  {
-    day: 31,
-    month: "December",
-  },
-];
+}
 
 export const daysToMonths = (days: Days, start?: StartForDaysToMonths) => {
   if (start === undefined) {
-    return (days / 30.4167) as Months;
+    return (days / 30.4375) as Months;
```

Comment:
if you don't specify `start` by default I use `30.4375` instead of `30.4167` due to average number of days in the year

```js
365.25 / 12 // 30.4375 ✅
```

So it's synced with `daysToYears` now 🔥